services/stream-functions/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "stream-functions",
   "description": "",
   "version": "1.0.0",
-  "dependencies": {
-    "dynamo-to-elasticsearch-nodejs": "^1.0.5",
-    "dynamodb-stream-elasticsearch": "^2.11.1"
-  },
-  "devDependencies": {}
```

Comment:
The devDependencies portion of this merge is incorrect.
The devDependencies value should not be changed for this PR; it should be as it was, {}.